### ISSUE TITLE
[New request] Garbage collection card with local calendars?

### ISSUE THREAD

**Issue by woody4165** [RedFoxy/HA-Separate-Garbage-Collection#6]
Hi and thanks for sharing this card

Since the Garbage collection project will be no more updated, are you going to move (or create) this project t use local calendars?

Or just suggest how to adapt it?

Thanks

**Comment by RedFoxy** [comment]
Yes, Just now I'm a bit too busy to finish and release next version

**Comment by pilot1981** [comment]
I

> Hi and thanks for sharing this card
> 
> Since the Garbage collection project will be no more updated, are you going to move (or create) this project t use local calendars?
> 
> Or just suggest how to adapt it?
> 
> Thanks

you found a solution?

**Comment by RedFoxy** [comment]
> you found a solution?

Still no news from home assistant group

**Comment by RedFoxy** [comment]
And the new version is here!

**Comment by RedFoxy** [comment]
New vewrsion is here!
